go/vt/vttablet/onlineddl/vrepl/unique_key.go

```diff
@@ -32,6 +32,12 @@ func UniqueKeyValidForIteration(uniqueKey *UniqueKey) bool {
 		// Thus, we cannot use this unique key for iteration.
 		return false
 	}
+	if uniqueKey.HasSubpart {
+		// vreplication does not fully supports partial keys, such as:
```

Comment:
Annoyingly nitty, but:
```suggestion
		// vreplication does not fully support indexes on column prefixes such as:
```

https://dev.mysql.com/doc/refman/en/create-index.html#create-index-column-prefixes